### ISSUE TITLE
[stable/grafana] Cant do deployment due to busybox error

### ISSUE THREAD

**Issue by charlspjohn** [helm/charts#12641]
**Is this a request for help?**: Yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

While deploying Iam getting the below error 

```
grafana-7fd948c87d-5lt7g   0/1   Init:ImagePullBackOff   0     3m32s

Events:
  Type     Reason                  Age                  From                                  Message
  ----     ------                  ----                 ----                                  -------
  Warning  FailedScheduling        113s (x3 over 118s)  default-scheduler                     pod has unbound immediate PersistentVolumeClaims (repeated 2 times)
  Normal   Scheduled               113s                 default-scheduler                     Successfully assigned grafana/grafana-7fd948c87d-5lt7g to aks-monitoringpr-23091975-0
  Normal   SuccessfulAttachVolume  62s                  attachdetach-controller               AttachVolume.Attach succeeded for volume "pvc-ae2cb34e-5178-11e9-933d-228fa2d976a3"
  Normal   SandboxChanged          32s (x2 over 35s)    kubelet, aks-monitoringpr-23091975-0  Pod sandbox changed, it will be killed and re-created.
  Normal   Pulling                 17s (x2 over 37s)    kubelet, aks-monitoringpr-23091975-0  pulling image "busybox:1.3"
  Warning  Failed                  16s (x2 over 36s)    kubelet, aks-monitoringpr-23091975-0  Failed to pull image "busybox:1.3": rpc error: code = Unknown desc = Error response from daemon: manifest for busybox:1.3 not found: manifest unknown: manifest unknown
  Warning  Failed                  16s (x2 over 36s)    kubelet, aks-monitoringpr-23091975-0  Error: ErrImagePull
  Normal   BackOff                 1s (x5 over 33s)     kubelet, aks-monitoringpr-23091975-0  Back-off pulling image "busybox:1.3"
  Warning  Failed                  1s (x5 over 33s)     kubelet, aks-monitoringpr-23091975-0  Error: ImagePullBackOff
```

**Version of Helm and Kubernetes**: 1.12

**Which chart**: stable/grafana

**What happened**: imagepullbackoff

**What you expected to happen**: to start grafana pod

**How to reproduce it** (as minimally and precisely as possible): helm del --purge && helm install --name grafana stable/grafana

**Anything else we need to know**:

**Comment by maorfr** [comment]
This should have been fixed by #12611.
Can you try using the newest chart version?

**Comment by charlspjohn** [comment]
Dear @maorfr
Still the deployment fails. I took the lines into my values file and encapsulated within single quotes (tag: '1.30') and completed the deployment
